src/ui/camera.js

```diff
@@ -950,6 +972,16 @@ class Camera extends Evented {
      *
      * @memberof Map#
      * @returns {FreeCameraOptions} The camera state
+     * @example
+     * const camera = map.getFreeCameraOptions();
+     *
+     * const position = [138.72649, 35.33974];
+     * const altitude = 3000;
+     *
+     * camera.position = mapboxgl.MercatorCoordinate.fromLngLat(position, altitude);
+     * camera.lookAtPoint([138.73036, 35.36197]);
+     *
+     * map.setFreeCameraOptions(camera);
```

Comment:
this example should be for `getFreeCameraOptions` it looks like

Reply:
I've chosen to be thorough there, explicitly making full use of the free camera API through getFreeCamera/update options/setFreeCamera, I think it's best to leave as is, so someone who has not been exposed to the entire API gets exposed to the full cycle.

Removing it means we end up having a `camera.lookAtPoint`, but where does the `camera` variable comes from? 

Best to leave as is IMO.